sphinx_automodapi/automodsumm.py

```diff
@@ -425,8 +427,6 @@ def generate_automodsumm_docs(lines, srcfn, app=None, suffix='.rst',
 
     from .utils import find_autosummary_in_lines_for_automodsumm as find_autosummary_in_lines
 
-    logger = logging.getLogger(__name__)
```

Comment:
I think it's better to keep those variables when the logger is used more than once, instead of calling `logging.getLogger(__name__).xxx` each time.

Reply:
Sorry, I thought that's what you meant by "put the `logging.getLogger` call in each function when it is needed."

Typically there's a single logger object per module. Is that what you'd prefer? Usually you would only create multiple logger  objects in function scope if you want to append the function name to the logger.

Reply:
Sorry for the confusion, I meant one logger per function/method as was already the case for functions already using `logging`.